internal/venice-common/src/main/java/com/linkedin/venice/meta/Partition.java

```diff
@@ -102,13 +128,23 @@ public List<Instance> getInstancesInState(String state) {
   public List<Instance> getReadyToServeInstances() {
```

Comment:
Unrelated to your change but ... the JavaDoc says remove this API once we are fully on L/F.. do you think we could remove it or it is the doc has problem?

Reply:
I cleaned this up.